docker/importer/importer_test.py

```diff
@@ -821,7 +825,7 @@ def setUp(self):
     self.tmp_dir = tempfile.mkdtemp()
 
     tests.mock_datetime(self)
-    warnings.filterwarnings("ignore", "unclosed", ResourceWarning)
+    warnings.filterwarnings('ignore', category=SystemTimeWarning)
```

Comment:
I didn't realize there was a way to squelch these, this is awesome! Thank you!!!